dashboard/config/routes.rb

```diff
@@ -599,6 +599,7 @@
     match '/lti/v1/login(/:platform_id)', to: 'lti_v1#login', via: [:get, :post]
     post '/lti/v1/authenticate', to: 'lti_v1#authenticate'
     match '/lti/v1/sync_course', to: 'lti_v1#sync_course', via: [:get, :post]
+    post '/lti/v1/create_integration', to: 'lti_v1#create_integration'
```

Comment:
Do we want to be slightly more RESTful and put this at `/lti/v1/integrations`? Opens us up to easier implementation of related routes in the future, e.g. GET /integrations, GET /integrations/:id etc.

Reply:
Yes, good call.